examples/pytorch/gin/multi_gpu_graph_prediction.py

```diff
@@ -118,21 +114,14 @@ def train(rank, world_size, dataset_name, root):
             loss.backward()
             optimizer.step()
         scheduler.step()
-        tt = time.time()
 
         if rank == 0:
-            print(tt - t0)
-        durations.append(tt - t0)
-
-        if rank == 0:
-            val_metric = evaluate(valid_dataloader, rank, model, evaluator)[evaluator.eval_metric]
-            test_metric = evaluate(test_dataloader, rank, model, evaluator)[evaluator.eval_metric]
+            val_metric = evaluate(valid_dataloader, rank, model.module, evaluator)[evaluator.eval_metric]
```

Comment:
Is it sufficient to pass `model`?

Reply:
No, according to my test, use DDP model directly may cause stuck here, so I change it to literal model.